Squadrosu.Framework/Board.cs

```diff
@@ -11,21 +14,51 @@ namespace Squadrosu.Framework;
 /// </summary>
 public sealed class Board
 {
+    /// <summary>
+    /// List of White Pieces
+    /// </summary>
+    public Piece[] White { get; private set; } = new Piece[5];
+
+    /// <summary>
+    /// List of Black Pieces
+    /// </summary>
+    public Piece[] Black { get; private set; } = new Piece[5];
+
+    /// <summary>
+    /// The board itself, and positions of pieces
+    /// </summary>
+    public Piece?[,] Positions { get; private set; } = new Piece[7, 7];
```

Comment:
Je suggère un truc plus descriptif
```suggestion
    /// <summary>
    /// Grid where pieces are placed and moved
    /// </summary>
    public Piece?[,] Positions { get; private set; } = new Piece[7, 7];
```
D'ailleurs c'est une bonne idée d'avoir fait une grille